Open_Foam/Post_Process.ipynb

```diff
@@ -4,129 +4,112 @@
-    "#### 5- Conclude the post-processing and validation with a last conclusive paragraph highlighting the pros and cons of the CFD simulations.\n",
-    "#### 6- Cite the source(s) that you have used for writing the content of your Domain file considering the copy right rules.  Make sure to cite the sources you use or refer to within the body of the text as well as in the end!\n",
+    "Courant Number mean: 0.0238165 max: 0.565953\n",
+    "```\n",
     "\n",
-    "# Handy Commands for Writing the Physics file: \n",
+    "The residual value of each variable denotes how the numerical value of that variable is differing from the previous time step. Lower value of residual indicates the obtained numerical value of the variable of interest has converged to a specific value. In this simulation this is true for `U_x`, `U_y` and `p` that have residual values by order of at least `1e-6`. The reported `continuity errors` defines how the obtained relevant variables are satisfying the continuity equation. The global continuity error with order of `1e-13` in this problem confirms that continuity equation is being satisfied.\n",
     "\n",
-    "## Insert a Figure\n",
+    "Once the simulation is ran in OpenFoam the user can make and plot the trend of evolution for different variables at the end of the simulations. Running the command `foamJob -s pimpleFoam` before starting the iterations and then in the end od simulation running the command `foamLog log` inside the main working directory will make a history of log files for different flow field variables inside a new directory called `logs`. Changing the directory into `logs` directory and running `gnuplot` command, user can plot the obtained numerical residuals. This process was done for simulation with $Re=100$ and $TSR=1$ and generated log files were visualized using the following commands:\n",
     "\n",
-    "```\n",
-    "<img src=\"./Images/sample.jpg\" width=\"500\">\n",
+    "```C++\n",
+    "gnuplot\n",
+    "gnuplot> set xlabel \"Iterations\"          #sets label of x-axis\n",
+    "gnuplot> set ylabel \"Residuals\"           #sets label of y-axis\n",
+    "gnuplot> set logscale y                   #sets logarithmic scale of y axis\n",
+    "gnuplot> plot './contLocal_0' with lines, './UxFinalRes_0' with lines, './pFinalRes_0' with lines #plots residuals\n",
+    "```   \n",
+    "\n",
+    "<img src=\"./Images/1residuals.png\" width=\"500\" align=\"middle\">\n",
     "</br>\n",
-    "Fig. 1 - The caption for the image goes here.\n",
+    "Fig. 1 The initial residual values of `U_x` and `p` with the continuity error values versus time steps.\n",
+    "\n",
+    "The general decreasing trend of the residual and error values confirm that the numerical solution of the conservation/governing equations are converging to specific final values. This decreasing trend is the first necessary, but not sufficient, general criteria to confirm the convergence of the CFD simulation.\n",
```

Comment:
Do you have history of residuals for a longer run of time steps? If yes, please include it. Furthermore, if time allows it will be great to add some discussion on observation of major spikes in the residual history due to the change of time step.